src/neo/SmartContract/Native/PolicyContract.cs

```diff
@@ -20,6 +20,7 @@ public sealed class PolicyContract : NativeContract
         private const byte Prefix_FeePerByte = 10;
         private const byte Prefix_BlockedAccounts = 15;
         private const byte Prefix_MaxBlockSize = 16;
```

Comment:
This should be `12`. Did you find the pattern?

Reply:
Got.